### ISSUE TITLE
textsum: poor quality of results after training - please help!

### ISSUE THREAD

**Issue by vgoklani** [tensorflow/models#920]
I built the model on top of 3.8M news articles, where the input was the first sentence, and the output was a manually hand-written headline (referred to as reference below). I ran the model for 1,000,000 steps, using a batch size of 4, and a beam-search of 8, on an AWS p2 instance for ~2 weeks. I have additional runs running in parallel on other instances, with larger batch sizes of 64 and 128 to compare. Increasing the batch size also increases the running time, as I *believe* the CPU isn't populating the queues fast enough. 

Here is the running_avg_loss from the last ten steps:

    999990. running_avg_loss: 2.103509
    999991. running_avg_loss: 0.719081
    999992. running_avg_loss: 7.949990
    999993. running_avg_loss: 0.664588
    999994. running_avg_loss: 2.355664
    999995. running_avg_loss: 3.864816
    999996. running_avg_loss: 0.270447
    999997. running_avg_loss: 0.716517
    999998. running_avg_loss: 0.514449
    999999. running_avg_loss: 1.110213
    1000000. running_avg_loss: 0.689220

There is still a lot of oscillation in the loss, should I continue training the model? Should I expect the results to be smoother?

Here are some examples of the output, note the results are terrible :(

0: article: trade between south korea and its communist neighbor north korea doubled last year , totalling more than one billion dollars for the first time , official data showed .
	reference: inter-korean trade doubles to one billion dollars in ####
	generated: shanghai summit at german street in first quarter

1: article: china has begun a push to promote small , low-emission cars as oil prices remain high and environmental concerns grow over the nation 's fast-rising auto culture , state press reported thursday .
	reference: china pushes smaller cars as environmental concerns grow
	generated: china to spend #.# bln chinese dlr chinese dollar in ####

2: article: finance minister ehud olmert will chair an emergency meeting of the israeli cabinet on thursday after prime minister ariel sharon suffered a massive stroke , public television reported .
	reference: olmert to chair emergency israel cabinet meet
	generated: pm 's socialist leader wo up new new

3: article: four-time all-star alexander mogilny was waived by new jersey on wednesday , one day after the devils welcomed back czech forward patrik elias .
	reference: mogilny odd man out as devils welcome back elias
	generated: indonesian islamic militants are three of new league

4: article: trade between south korea and its communist neighbor north korea doubled last year , totalling more than one billion dollars for the first time , official data showed .
	reference: inter-korean trade doubles to one billion dollars in ####
	generated: shanghai summit at german street in first quarter

5: article: gunmen shot dead seven tribesmen thursday in a restive tribal region along the pakistan-afghanistan border where the military is hunting al-qaeda-linked militants , officials said .
	reference: seven tribesmen shot dead in pakistani tribal area
	generated: more policeman in <unk> death

6: article: oil prices were slightly easier in asian trade thursday , marking time ahead of key us stockpile figures and after resolution of the russia-ukraine gas pricing dispute , dealers said .
	reference: oil prices were easier in asian trade
	generated: oil prices in bombay morning #.## years

7: article: japanese share prices rose #.## percent in morning trade thursday to hit the highest level in more than five years as fresh gains on wall street fanned upbeat investor sentiment here , dealers said .
	reference: tokyo shares rise #.## percent in morning trade
	generated: trade prices in tokyo close at #.# percent lower

8: article: us president george w. bush said late wednesday that he and the first lady shared `` the concerns of the israeli people about prime minister ariel sharon 's health '' and were praying for his recovery .
	reference: bush says he shares israelis concern over sharon
	generated: us envoy concerned about iraq

9: article: right fielder jeromy burnitz inked a one-year , #.# million-dollar contract with the pittsburgh pirates on wednesday , a week after he was reported as close to a deal with baltimore .
	reference: pirates ink outfielder burnitz to one-year deal
	generated: two french workers to reach opposition 's police change

**Comment by asimshankar** [comment]
This question is better suited for [stackoverflow](http://stackoverflow.com/questions/tagged/tensorflow) as we try to keep the github issues focused on bugs and feature requests.
